e2e_tests/tests/fixtures/core_api/non_conforming.py

```diff
@@ -0,0 +1,23 @@
+import determined as det
+
+
+# NEW: given a checkpoint_directory of type pathlib.Path, save our state to a file.
+# You can save multiple files, and use any file names or directory structures.
+# All files nested under `checkpoint_directory` path will be included into the checkpoint.
+def save_state(x, steps_completed, trial_id, checkpoint_directory):
```

Comment:
actually delete this whole function, it looks unused.

Reply:
oh yeah, oops.